<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="158" started-at="2014-12-23T08:21:38Z" finished-at="2014-12-23T08:21:38Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="158" started-at="2014-12-23T08:21:38Z" finished-at="2014-12-23T08:21:38Z">
      <class name="testCases.BuyPositiveTC1">
        <test-method status="FAIL" signature="beforeMethod()[pri:0, instance:testCases.BuyPositiveTC1@14f1544]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2014-12-23T13:51:38Z" finished-at="2014-12-23T13:51:38Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at utility.ExcelUtils.getCellData(ExcelUtils.java:19)
	at utility.Utils.OpenBrowser(Utils.java:24)
	at testCases.BuyPositiveTC1.beforeMethod(BuyPositiveTC1.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:653)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="SKIP" signature="mainTest()[pri:0, instance:testCases.BuyPositiveTC1@14f1544]" name="mainTest" duration-ms="0" started-at="2014-12-23T13:51:38Z" finished-at="2014-12-23T13:51:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mainTest -->
      </class> <!-- testCases.BuyPositiveTC1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
